src/lando/dockerflow/views.py

```diff
@@ -0,0 +1,83 @@
+from django.db import connections
+from django.db.utils import OperationalError
+from django.http import JsonResponse
+from django.views import View
+
+from lando.dockerflow.decorators import disable_caching, log_request
+
+
+class VersionView(View):
+    """
+    This view handles the version information of Lando.
+
+    It returns a JSON response containing the version information.
+    """
+
+    @disable_caching
+    @log_request
+    def dispatch(self, request, *args, **kwargs):
+        return super().dispatch(request, *args, **kwargs)
```

Comment:
This should be defined at the top in a base class, say `DockerflowView`, and each other view should just extend that base class.

Reply:
Done.